src/pages/DetailPage/DetailPageView.tsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from 'react-router-dom';
+import { DetailsPageProps } from '../../types/interfaces';
+
+const DetailsPage: React.FC<DetailsPageProps> = (props) => {
+  console.log(props);
+  const handleBackClick = () => {
+    props._updateSelectedPlanet();
+  };
+  return (
+    <div className="details-container">
+      <div>
+        <h2>Details</h2>
+        {props.selectedPlanet ? (
+          <>
+            <p>Name: {props.selectedPlanet.name}</p>
+            <p>Climate: {props.selectedPlanet.climate}</p>
+            <p>Diameter: {props.selectedPlanet.diameter}</p>
+            <p>Rotation Period: {props.selectedPlanet.rotation_period}</p>
+            <p>Terrain: {props.selectedPlanet.terrain}</p>
+          </>
+        ) : (
+          <p>No details available</p>
+        )}
+        <Link to={`/search/${props.currentPage}`} onClick={handleBackClick}>
+          Back to Results
+        </Link>{' '}
```

Comment:
maybe you don't need  it))

Reply:
yes, it's deleted

Reply:
and write this 
        <button onClick={handleBackClick}>Back to Results</button>